### ISSUE TITLE
Metadata Widget Enhancement and issues

### ISSUE THREAD

**Issue by Pooya-Oladazimi** [ts4nfdi/terminology-service-suite#108]
Some enhancement suggestions for the metadata widget

https://ts4nfdi.github.io/terminology-service-suite/comp/latest/?path=/docs/react_metadatawidget--docs

- The term label should be bold and on top
- The tabs should be optional. Users can decide which tab to show
- Backlink to the TS: The term should be a link (if the user wants) to the corresponding terminology service. (show the target term in the target TS)
- **Question**: **what if the user only has an Iri? (type is unknown).** 

Issues:
The doc seems to be not correct. It looks like a search doc. Term fetching in OLS does not have these options:
![Screenshot from 2024-07-10 10-40-24](https://github.com/ts4nfdi/terminology-service-suite/assets/16546640/c4687aef-8631-4d87-b48d-746abf4be4ef)

Types options have strange values (empty and invalid string)
![Screenshot from 2024-07-10 10-42-42](https://github.com/ts4nfdi/terminology-service-suite/assets/16546640/8d44369a-58ce-47a9-a8bf-1b7a35da8eb4)

**Comment by VincentKneip** [comment]
I agree that the entity label should be bold and moved over the iri. The breadcrumb should stay on top in my opinion, similar to the appearance on EBI-OLS: https://www.ebi.ac.uk/ols4/ontologies/hp/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FHP_0000819

Regarding your question: If the `entityType` is unknown, the empty string option should be active on the `entityType` property. This has the same effect as passing `undefined` to the property, and leads to the widget figuring the correct value of `entityType` out by itself.

The value `INVALID STRING` was introduced for testing purposes on incorrect input values. When selected, an error is thrown on the console: 
![grafik](https://github.com/user-attachments/assets/1e1c087d-2d80-4318-90d3-4b955eda7986)

You're right with the docs for the `parameter` property. Currently, all widgets having the `parameter` property incorrectly have a doc for search parameters. This has to be corrected for all widgets not using the search endpoint.
